### PR TITLE
Adding 2.5.0 / 2.5.1 / 2.6.0 / 1.3.8/ 1.3.9 for perf-test

### PR DESCRIPTION
**Pull request by jordarlu**
### Description
Adding 2.5.0 / 2.5.1 / 2.6.0 / 1.3.8/ 1.3.9 for perf-test 

### Issues Resolved
This PR is for adding 2.5.0 / 2.5.1 / 2.6.0 / 1.3.8/ 1.3.9 for perf-test to execute 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by peterzhuamazon** [comment]
Do we still need to keep the ones that is already out?
2.2.1/2.3.0, etc.

Thanks.

File: jenkins/opensearch/perf-test.jenkinsfile [hunk 1]
```diff
@@ -28,14 +40,22 @@ pipeline {
             H 17 * * * %BUNDLE_MANIFEST_URL=https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.4.0/latest/linux/arm64/tar/dist/opensearch/manifest.yml;TEST_ITERATIONS=250;WARMUP_ITERATIONS=0
```
Comment (bbarani) [inline jenkins/opensearch/perf-test.jenkinsfile:40]: I think we can remove older versions (i.e <2.5.0 and 1.3.8)

**Comment by jordarlu** [comment]
> Do we still need to keep the ones that is already out? 2.2.1/2.3.0, etc.
> 
> Thanks.

agh.. good question ... I think it makes sense to remove those already out .... also, easier to find a time slot between each build to run ... any objections from team?

File: jenkins/opensearch/perf-test.jenkinsfile (hunk 1 above)
Comment (jordarlu) [reply jenkins/opensearch/perf-test.jenkinsfile:40]: Thank you @bbarani and @peterzhuamazon ... cool .. let me remove those already out ...

**Comment by jordarlu** [comment]
@peterzhuamazon , @bbarani , I have got older versions removed. Thanks in advance for your 2nd review !

**Review by bbarani — APPROVED** [review]

**Pull request merged**